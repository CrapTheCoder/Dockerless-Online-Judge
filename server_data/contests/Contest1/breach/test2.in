740210442065324019 786545855384565934