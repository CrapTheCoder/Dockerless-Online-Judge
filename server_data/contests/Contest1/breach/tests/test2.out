1526756297449889953